ciphey/basemods/Decoders/baconian.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional, Dict, List
+from ciphey.iface import ParamSpec, Config, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class Baconian(Decoder[str, str]):
+
+    lookup = {'A':'aaaaa', 'B':'aaaab', 'C':'aaaba', 'D':'aaabb', 'E':'aabaa', 
+        'F':'aabab', 'G':'aabba', 'H':'aabbb', 'I':'abaaa', 'J':'abaab', 
+        'K':'ababa', 'L':'ababb', 'M':'abbaa', 'N':'abbab', 'O':'abbba', 
+        'P':'abbbb', 'Q':'baaaa', 'R':'baaab', 'S':'baaba', 'T':'baabb', 
+        'U':'babaa', 'V':'babab', 'W':'babba', 'X':'babbb', 'Y':'bbaaa', 'Z':'bbaab'}
+    
+    lookup_default = {'A':'aaaaa', 'B':'aaaab', 'C':'aaaba', 'D':'aaabb', 'E':'aabaa', 
+        'F':'aabab', 'G':'aabba', 'H':'aabbb', 'I':'abaaa', 'J':'abaaa', 
+        'K':'abaab', 'L':'ababa', 'M':'ababb', 'N':'abbaa', 'O':'abbab', 
+        'P':'abbba', 'Q':'abbbb', 'R':'baaaa', 'S':'baaab', 'T':'baaba', 
+        'U':'baabb', 'V':'baabb', 'W':'babaa', 'X':'babab', 'Y':'babba', 'Z':'babbb'}
+
+
+
+    def decode(self, ctext: str) -> Optional[str]:
+        """
+        Performs Baconian decoding
+        """
+        i = 0
+        
+        default_decoced_string = ""
+        decoced_string = ""
+
+        return_default_string = True
+        while True:
```

Comment:
Can you confirm 100% that this will break every single time? :) I would rather not have `while True` because it can lead to undefined behaviour :)